<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWidget</class>
 <widget class="QWidget" name="CMainWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BigAssClock</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/main</normaloff>:/icons/main</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/pin</normaloff>:/icons/pin</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAbout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSettings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelCurrentHour">
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarHour">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNextHour">
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelCurrentMinute">
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarMinute">
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNextMinute">
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelCurrentSecond">
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarSecond">
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNextSecond">
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelDate">
     <property name="text">
      <string>18 Dec 2022
Sunday</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>CMainWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWidget</receiver>
   <slot>setAlwaysOnTop(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setAlwaysOnTop(bool)</slot>
 </slots>
</ui>
